packages/expo-video/src/VideoView.web.tsx

```diff
@@ -12,6 +12,7 @@ class VideoPlayerWeb implements VideoPlayer {
   isPlaying: boolean = false;
   _isMuted: boolean = false;
   timestamp: number = 0;
+  volume: number = 1;
```

Comment:
If there is a way to get the volume on web, can you follow up on this? (in a separate PR)